docs/reference-architectures/data/monitoring.md

```diff
@@ -0,0 +1,147 @@
+---
+title: Solution Accelerator for Monitoring Azure Databricks
+titleSuffix: Azure Solution Accelerators
+description: A scala library to deploy to a Databricks cluster to enable monitoring of metrics and logging data in Azure Log Analytics
+author: petertaylor9999
+ms.date: 01/28/2019
+ms.topic:
+ms.service:
+ms.subservice:
+---
+
+# Solution Accelerator: Monitoring Azure Databricks using Azure Log Analytics
+
+Azure Databricks is a fast, easy and collaborative Apache Spark–based analytics service that makes it easy to rapidly develop and deploy big data analytics and artificial intelligence (AI) solutions. Monitoring is a critical part of any production-level solution, and Azure Databricks offers robust functionality for monitoring metrics, streaming query event information, and application logging data.
+
+Databricks is a flexible and extensible service that supports delivery of metrics and logging data to many different logging services. This solution accelerator extends the core monitoring functionality of Azure Databricks to support Azure Log Analytics.
+
+# Monitoring Apache Spark Structured Streaming queries in Azure Databricks
+
+Azure Databricks is based on Apache Spark, which includes a set of structured APIs for batch processing data using Datasets, DataFrames, and SQL. With Apache Spark 2.0, support was added for [Structured Streaming](https://spark.apache.org/docs/latest/structured-streaming-programming-guide.html), a data stream processing API built upon Spark's batch processing APIs.
+
+The Structured Streaming API includes functionality to monitor active streams as data is processed by sending event information to an external metrics system. The external metrics system is registered with Spark Structured Streaming by attaching an implementation of a **StreamingQueryListener** object to the current Spark streaming session.
+
+This solution accelerator includes a **StreamingQueryListener** implementation, **LogAnalyticsStreamingQueryListener**, to send Structured Streaming event information to an Azure Log Analytics workspace. You first attach the **LogAnalyticsStreamingQueryListener** to the Spark session in your job application code. Next, the solution accelerator includes a Spark initialization script that copies the **LogAnalyticsStreamingQueryListener** implementation to each node in your Spark cluster. Finally, specify the connection string, workspace ID, and shared access signature (SAS) token for your Log Analytics workspace in the Spark configuration settings. When your job runs, metrics are sent to the specified Log Analytics workspace.
+
```

Comment:
There is a lot of interactive information in this paragraph. An image would be hugely helpful for me to determine what is it that I am doing, what is the solution accelerator doing, etc.